### DISCUSSION TITLE
Flsun QQ-S Pro Sherpa Mini Direct Drive

### DISCUSSION THREAD

**Discussion by sokannsgehn** [blackfyre/flsun.community#98] [Q&A]
Hallo zusammen,
Jetzt hab ich hier immer soviel interessantes gelesen und gesehen und bin dann zu dem Entschluss gekommen, meinen QQ-Pro mit dem Sherpa Mini Direct Drive auszustatten.
Hab diverse Adapter runtergeladen und ohne Problem zusammenbauen können. Nachdem ich ja einen anderen Stepper Motor habe, musste ich ja auch diesen Kalibrieren. Ich hab das vorher schon beim original Motor gemacht und musste ihn in der Firmware robin_mini_config ändern.
 >EXT0_STEPS_PER_MM		     394.20			
>EXT1_STEPS_PER_MM		     394.20
Jetzt hab ich das mit dem  Sherpa Mini Extruder Hotend 
( https://www.amazon.de/dp/B09B24ZRT5?psc=1&ref=ppx_yo2ov_dt_b_product_details )
wieder gemacht und komm jetzt auf einen Wert 
EXT0_STEPS_PER_MM		      1163,51			

EXT1_STEPS_PER_MM		      1163,51
kann das stimmen, oder muss ich da irgendeinen Faktor umstellen/umschreiben.
Das extrudieren war genau 99,4mm, also fast perfekt, ich hab jetzt nur Bammel das ich da was zerstöre, wenn es nicht richtig eingestellt ist.
Bitte um Hilfe, hab auch erst vor knapp 2 Monaten überhaupt mit 3d Drucken angefangen, also bitte bringt mir es bitte schonend bei.

**Comment by blackfyre** [comment]
@sokannsgehn 

Welcome to Github!

I'm sorry to say that the issues section on this repository is reserved for actual issues with the site. Not issues arising from changing parts of your printer.
Hence, I'm converting this issue into a discussion.
